azurelinuxagent/common/protocol/wire.py

```diff
@@ -967,24 +1052,36 @@ def send_request_using_appropriate_channel(self, direct_func, host_func):
 
         return ret
 
-    def upload_status_blob(self):
-        ext_conf = self.get_ext_conf()
+    def get_extensions_fast_track_enabled(self):
+        # Eventually we'll need to generalize this method to write all features
+        # but for now we only have one
+        fast_track_enabled = False
+        if conf.get_extensions_fast_track_enabled():
+            if self.hostgaplugin_supports_fast_track():
+                logger.periodic_info(logger.EVERY_DAY, "FastTrack enabled")
```

Comment:
Those messages should probably not be periodic

Reply:
These ones I argue should be OK. If anything, they should be less often than daily.
- This method is called every 3 seconds
- The state is highly unlikely to change. It would require downgrading the HostGAPlugin build.